schedview/compute/multisim.py

```diff
@@ -0,0 +1,366 @@
+import itertools
+import math
+
+import numpy as np
+import pandas as pd
+
+
+def often_repeated_fields(visits: pd.DataFrame, min_counts: int = 4):
+    """Find often repeated fields in a table of visits.
```

Comment:
Can you make some comment in the docstring or in the code about what this is for? Why four visits to a given field for a "often visited field"? 
In operations sims, four visits to a field would be something that was either a DDF, a near-sun twilight microsurvey visit, or a field which was caught during twilight and again during the night .. we shouldn't generally be covering other fields more than that.  In auxtel sims now, four visits to a field is not uncommon. 
Between these two use-cases, I'm confused what this is for.

Reply:
Yes, my intention was to exclude most WFD fields, but include fields where we spend a lot of time.
The idea is to provide a single table for people interested in specific fields without producing something too long to be convenient. That is, I expect that it will be much more common for a user to want to know "how many times are we likely we to observe this specific DDF field tonight" than it is for a user to want to know about any specific WFD field.
We might want something to do something similar for larger areas of WFD coverage, and I have a few ideas, but none I'm really happy with. This is something to be added later.